### PR TITLE
[DDW-659] UI - Delegation setup wizard step 2

### PR DESCRIPTION
**Pull request by tomothespian**
This PR introduces Delegation Setup wizard step 2 UI.

## Todos

- [x] Add Japanese translations
- [x] Integrate stake pool tooltip logic once is done in DDW-716 (https://github.com/input-output-hk/daedalus/pull/1431)
- [x] Merge Delegation steps wizard 0 & 1 into develop which is base for this step (STEP 2)
- [x] Merge Stake pool tooltip positioning branch once is done and test (https://github.com/input-output-hk/daedalus/pull/1429)

## Screenshots

_Choose stake pool dialog - regular_
Choose stake pool dialog - regular

Choose stake pool dialog - hovered thumb
Choose stake pool dialog - hovered thumb

Choose stake pool dialog - selected pool
Choose stake pool dialog - selected pool

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1562143494198700)
- [ ] Test Storybook story under `Staking >DelegationStepsChooseStakePoolDialog`
- [ ] Test Storybook story under `Staking >Stake Pools` _(check if tooltips are shown as is expected after changes on shared component)_

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by IuliaDolishniak** [comment]
@tomothespian should the color of tooltip be changed depending on the theme selected?

**Comment by tomothespian** [comment]
> @tomothespian should the color of tooltip be changed depending on the theme selected?

@IuliaDolishniak I used colors Danilo defined for tooltip component but with additional transparency as is defined. Maybe @a-rukin can help us here and define if colors should be different.

**Comment by IuliaDolishniak** [comment]
I can see that @daniloprates colors of tooltips are different for every theme selected.

**Comment by tomothespian** [comment]
> I can see that @daniloprates colors of tooltips are different for every theme selected.

I added transparency on default colors and can change that once @a-rukin define proper colors.

**Comment by a-rukin** [comment]
image

this cropping confusing me =/ 


image

let's have this area fixed 
67px height = top margin + header height + step number height

**Comment by tomothespian** [comment]
> image
> 
> this cropping confusing me =/
> 
> image
> 
> let's have this area fixed
> 67px height = top margin + header height + step number height

@a-rukin it is done, can you re-check please?

**Comment by tomothespian** [comment]
@a-rukin Stake pool tooltips in Delegation step dialog is reverted back to its default styles but without "Delegate" button as you suggested (https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1562143494198700).
Can you also check that?

**Comment by a-rukin** [comment]
imageimage 
text position (or size) is different

**Comment by IuliaDolishniak** [comment]
@a-rukin @tomothespian please advise the color of the selected stake pool shouldn't be of the same color as the stake pool is?
screen

**Comment by IuliaDolishniak** [comment]
The word "Pool" shouldn't be translated into Japanese?
screen

**Comment by a-rukin** [comment]
@IuliaDolishniak yeah, you are right, already told this to @tomothespian

File: source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseStakePoolDialog.js [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+  currentTheme: string,
+  onClose: Function,
+  onContinue: Function,
+  onBack: Function,
+};
+
+type State = {
+  searchValue: string,
+  selectedList?: ?string,
+  flipHorizontal: boolean,
```
Comment (daniloprates) [inline source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseStakePoolDialog.js]: The `flipHorizontal` and `flipVertical` state vars are not needed anymore. I'll remove them.

**Comment by a-rukin** [comment]
@daniloprates can we have time out only for tooltip, not for glow? 
+ sometimes it blocks clicking =/

File: source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseStakePoolDialog.js (hunk 1 above)
Comment (tomothespian) [reply source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseStakePoolDialog.js]: @daniloprates nice work and thanks 🥇

**Comment by gnpf** [comment]
aMHbOlyfu9
@tomothespian On this dialog, the pools are displayed on two rows on Japanese, while they fit in just one in English.

Also, sometimes when switching between Japanese and English, some of the text stays on the previous language. It's visible on the previous gif, but I'm copying another example after this. I'm assuming this is maybe a cache issue or something related to Storybook because it shows the correct text after refreshing, but I just wanted to mention it in case there is something else going on.

image

**Comment by tomothespian** [comment]
@gnpf Related to translations you saw in storybook ("steps indicator label"). I already explained that to QA team. Translations are managed in parent component and can't be handled on that way because specific child components are presented in storybook. I added manual translations handler to storybook so you can simulate and see that also "Steps indicator label" is translated but can't handle that right now automatically without refresh before [DDW-711](https://github.com/input-output-hk/daedalus/pull/1426) is merged.

I will check weird stake pool block positions.

**Comment by tomothespian** [comment]
@gnpf Changes you requested are done! Can you please re-check?

**Pull request merged**